docs/integrations/keras.rst

```diff
@@ -1,112 +1,319 @@
+   model.add(Dense(200, activation='relu'))
+   model.add(Dropout(PARAMS['dropout']))
+   model.add(Dense(100, activation='relu'))
+   model.add(Dense(1, activation='sigmoid'))
+
+   # compile the model
+   model.compile(optimizer=PARAMS['optimizer'], loss=PARAMS['loss'], metrics=[PARAMS['metric']])
+
+   # summarize the model
+   print(model.summary())
+
+
+.. image:: ../_static/images/integrations/keras-integration/model-summary.png
+
+
+
+The model is pretty basic, and uses an |embedding| layer as the input because you're working with text inputs of large dimensions. The output layer is a sigmoid node, because this is a binary classification problem (Spam or Not-Spam).
+In the next section, you'll start model training as you normally would when training a keras model, with just one exception in the callbacks section.
+
+.. |embedding| raw:: html
+
+    <a href="https://machinelearningmastery.com/use-word-embedding-layers-deep-learning-keras/#:~:text=2.-,Keras%20Embedding%20Layer,API%20also%20provided%20with%20Keras." target="_blank">embedding</a>
+
+
+.. code-block:: python3
+
+   # fit the model
+   model.fit(x=padded_train,
+            y=y_train,
+            epochs=PARAMS['epochs'],
+            batch_size=PARAMS['batch_size'],
+            validation_data=(padded_test, y_test), verbose=1,
+            callbacks=[NeptuneMonitor()]
+            )
+
+.. image:: ../_static/images/integrations/keras-integration/training-out.png
 
-Usage
-^^^^^
 
-To log your experiments to Neptune, use the 
-:class:`~neptunecontrib.monitoring.keras.NeptuneMonitor` callback as an argument
-to the :meth:`keras.models.Model.fit` method and other Keras methods supporting
-training callbacks. An integration snippet is presented below.
+Notice that we pass the NeptuneMonitor() to the callbacks parameter of the model.fit method? 
+This is the one line integration of Neptune client with keras. 
+This integration code will send all training metrics and logs like epoch loss, epoch accuracy, batch loss and batch accuracy and so on, in real time, and this can be monitored in the dashboard of your |experiment|
+
+.. image:: ../_static/images/integrations/keras-integration/metrics-nep.png
+
+
+Click on |charts| to see live training logs
+
+Click on |logs| to download training logs
+
+Isn't it amazing how with just a single line of code you and your team can get to log and monitor metrics in real time? It sure is! Now you can start long running model training and have a Neptune effectively monitor and log all metrics for you, and with just the dashboard link, you can check your experiments on the go.
+
+.. |charts| raw:: html
+
+    <a href="https://ui.neptune.ai/risingodegua/spam-classifier/e/SPAM-4/charts" target="_blank">charts</a>
+
+
+.. |logs| raw:: html
+
+    <a href="https://ui.neptune.ai/risingodegua/spam-classifier/e/SPAM-4/logs" target="_blank">logs</a>
+
+
+
+EXTRA SECTION
+=============
+In this extra section, You'll:
+
+* Test your model
```

Comment:
This should containt:

- log testing metrics
- log charts after training 
- log model checkpoint after training
- log additional objects during training (extending NeptuneMonitor)